src/components/navbar/navDropdown/NavDropdown.module.scss

```diff
@@ -0,0 +1,60 @@
+@import '../../../utils/variables';
+
+.dropdownContainer {
+    z-index: 1;
+    background-color: $white;
+    position: absolute;
+    display: inherit;
+    top: 0;
+    left: calc(5vw - 10px);
+    width: 90vw;
```

Comment:
I wouldn't use `vw`. It's not commonly used, can make it harder to standardise things and not as clear as `px`. I don't see that much of a benefit of using it over `px`.

Reply:
Same as above. The benefit is responsiveness.